src/app/globals.css

```
@@ -1,4 +1,5 @@
 @import "tailwindcss";
+@variant dark (&:where(.dark, .dark *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)